### DISCUSSION TITLE
Override the sunset time with a fixed time.

### DISCUSSION THREAD

**Discussion by DrPerryCoke** [basnijholt/adaptive-lighting#348] [Q&A]
Greetings, I don’t understand a little how this function works, for example, if I set the sunset time to 19:00:00, should the brightness be at its minimum at this time?
Where does the time of sunrise and sunset come from if it is not assigned by me?
It would be nice for offset to set the time not only in seconds, for example, with the addition ```s```, ```m```, ```h```

**Comment by DrPerryCoke** [comment]
The fact is that I use the integration to control the light in the aquarium, I need 12 hours of light. The light should begin to shine at 7:00 and end at 19:00.
I partially managed to achieve this ...
Partially because during the day I had a maximum brightness of 70%, although it should be 100%
![screenshot](https://user-images.githubusercontent.com/76223634/198893451-b4dc2f69-f5a3-42fa-8202-dd6534fbf275.png)

**Comment by daFreeMan** [comment]
I'm not 100% certain, but your sunrise/sunset settings in the image really look off to me. 

Try setting your sunrise_time to `07:00:00` and sunset_time to `19:00:00`. Then remove the sunrise/sunset _offset settings (leave them blank) and see if that works for you.

AIUI, the way you've currently got it set is defining "sunrise" at 07:00:00, then offsetting that by 19,000 seconds which is 316 minutes (5.25 hours) which puts your defined "sunrise" at about `12:15:00`. 

Also you're defining "sunset" at `18:00:00` (6pm, not your desired 7pm), then offsetting by -20,000 seconds (333 minutes or 5.5 hours), which, by my math gives you a "sunset" of about `12:30:00`. i.e. your lights are on for about 15 minutes from roughly noon:15 to noon:30.

At least, that's how I read this. I've been known to be wrong before, though, so don't take this as gospel.
